datagateway_api/src/search_api/panosc_mappings.py

```diff
@@ -43,14 +44,14 @@ def get_icat_mapping(self, panosc_entity_name, field_name):
         :raises FilterError: If a valid mapping cannot be found
         """
 
-        log.info(
-            "Searching mapping file to find ICAT translation for %s",
-            f"{panosc_entity_name}.{field_name}",
-        )
+        # log.debug(
+        #    "Searching mapping file to find ICAT translation for %s",
+        #    f"{panosc_entity_name}.{field_name}",
+        # )
 
         try:
             icat_mapping = self.mappings[panosc_entity_name][field_name]
-            log.debug("ICAT mapping/translation found: %s", icat_mapping)
+            # log.debug("ICAT mapping/translation found: %s", icat_mapping)
```

Comment:
Was this a bit verbose? I can't remember if I found this too noisy when I ran the API. If so, feel free to remove this line (and other logging statement above) completely

Reply:
I presume that it was too verbose but I do not mind to put it back

Reply:
No you can remove this line. I do remember it was verbose when the line was added so I think this can be removed.